dbs/mongo/queries/markQuestion.js

```diff
@@ -0,0 +1,27 @@
+const {Questions} = require('../models.js');
+
+async function markQuestion(q_id, status){
+  if (!q_id && typeof q_id !== 'number') {
+    throw 'Invalide question id'
+  }
```

Comment:
Do you need same validation for markAnswer function?

Reply:
yes, added!